website/src/plugins/test-resourcecs/mock-release-data.ts

```diff
@@ -0,0 +1,1692 @@
+/**********************************************************************
```

Comment:
the folder name "test-resourcecs"

I would guess there is typo in the folder name ?

Reply:
Thanks for your attention :) It was a typo, I changed it to test/resources because test-resources was in the .gitignore